modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -12,51 +12,32 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 * 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
```

Comment:
@afelisatti should we adjust this values based on more recent performance tests?